api/tests/test_views.py

```diff
@@ -1,64 +1,153 @@
 import pytest
+import json
 
+from django.test import TestCase
+from django.utils import timezone
+from rest_framework.test import RequestsClient
 
-def test_fire():
-    """
-    Employees related test
-    """
-    pass
+from ..factories import EmployeeFactory, IncomeFactory, FutureEventFactory, TodaysEventsFactory, PastEventFactory
 
+SERVER_ADDRESS = 'http://127.0.0.1:8000'
+CLIENT = RequestsClient()
+EMPLOYEES = 'employees'
+EVENTS = 'events'
+INCOMES = 'income'
 
-def test_hire():
-    """
-    Employees related test
-    """
-    pass
 
+def _url_not_detail(content, method='', options=''):
+    return SERVER_ADDRESS + '/api/{}/{}/{}'.format(content, method, options)
 
-def test_change_position():
-    """
-    Employees related test
-    """
-    pass
 
+def _url_detail(content, id, method='', options=''):
+    return SERVER_ADDRESS + '/api/{}/{}/{}/'.format(content, id, method, options)
 
-def test_summary_cost_of_salaries():
-    """
-    Employees related test
-    """
-    pass
 
+def _option(option, value):
```

Comment:
This method could be probably removed. When using django test client there's usually param `data` where you can pass dict with keys & values. The client handles adding it to the url under the hood (in case of `GET` requests) - ref: https://docs.djangoproject.com/en/dev/topics/testing/tools/#django.test.Client.get

Reply:
done